.circleci/config.yml

```diff
@@ -125,12 +124,20 @@ jobs:
           name: Push Docker image to Quay
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS quay.io
-            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
             docker push quay.io/plotly/image-exporter:$CIRCLE_SHA1
+
+            # tag and push as branch name
+            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
```

Comment:
If `$CIRCLE_BRANCH` has the same name as an existing git tag, it will overwrite the existing image. This seems error-prone to me. Maybe we could tag the image `quay.io/plotly/image-exporter:branch-$CIRCLE_BRANCH`? See my next comment for the rationale behind this.

Reply:
Good point. Thanks!

> Maybe we could tag the image quay.io/plotly/image-exporter:branch-$CIRCLE_BRANCH? 

I'm a fan of this. Maybe on-prem devs have a strong opinion against it?